src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -24,4 +26,13 @@ public static boolean isEven(int a) {
     public static boolean isOdd(int a) {
         return !isEven(a);
     }
+
+    public static int countEven(int[] array) {
+        Preconditions.checkNotNull(array, "Your array is null!");
+        int evenElements = 0;
+        for (int e : array) {
+            if (isEven(e)) evenElements++;
```

Comment:
We should use curly braces with if, for and other statements. As an example see "[Google Java Style Guide: 4.1.1 Braces are used where optional](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)".

FYI: see also the reason why we should use curly braces: https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces

So, please,  use curly braces :)